### DISCUSSION TITLE
After uninstalling APatch, can the bootloader be locked again?

### DISCUSSION THREAD

**Discussion by jrmoore** [bmax121/APatch#465] [Q&A]
Let me elaborate a bit more, I have a device that currently has an unlocked bootloader, and while it is rooted, the software it is currently running is the official one (everything was Fastboot flashed).

Since I am using APatch, at least the boot image had to be modified, my questions are:
 - After flashing an unmodified boot image, would locking the bootloader be a possibility, or would I need to flash it all again?
 - Is there anything else changed during APatch's execution that would trip AVB? For example, `vbmeta` partitions.
 
I am asking because I am planning on locking the bootloader, but if at all possible I would like to use root access beforehand to disable (not uninstall) some system apps, since the default OS doesn't allow me to, and to restore a couple of app's data with Swift Backup. The `/system` partition would be untouched, same as `/vendor`, and as far as I know the `/data` one isn't signed with AVB, but I'd rather not end up with a device incapable of booting 😅

**Comment by marat2509** [comment]
In theory, it is enough to restore all (absolutely all) partitions that were modified (`boot` and `vbmeta` I suppose)
The `/system` etc. partitions themselves are not actually modified, but are only overlaid on top of them by OverlayFS (via APModules).
